website/docs/troubleshooting.md

```diff
@@ -36,3 +36,59 @@
     mixed refresh rate out of the box), if so,that's because X11 does not support mixed refresh
     rate well. You may be able to fix this through your compositor or by switching to wayland.
     As a temporary work around, you may set `g:neovide_refresh_rate` to the lower value.
+
+## Performance Profiling
+
+If you encounter a performance problem like frame rate stuttering, besides attaching a log file when
+reporting bugs, [tracy](https://github.com/wolfpld/tracy) profiling data will also be very useful and
+can usually help developers to troubleshoot the bug much faster. Here is how you can collect tracy data.
+
+1. Install tracy. Windows users can download it at
+[its GitHub release page](https://github.com/wolfpld/tracy/releases). Linux and macOS users can install
+it with package manager. Otherwise, you may have to build it yourself following tracy docs.
+
+2. Install [Rust SDK](https://www.rust-lang.org).
+
+3. Download or clone [source code of Neovide](https://github.com/neovide/neovide). Build it with
```

Comment:
I think we can remove the build instructions here. Since `cargo run` will automatically build, and it simplifies the steps needed.

Reply:
I did consider about this. But I worry that people wait for a long time of building after starting tracy and before starting neovide, which might be some kind wired (I think). So I instruct people to build Neovide here, so that they can first ensure Neovide is built successfully, and then just run a good Neovide after starting tracy.

I think I can add a note about this. And I don't think an extra step can bother too much.

Reply:
Emmm, now I think there are a lot of steps. It might stop people from doing this. I think I can squash some steps to make it neater.

Reply:
I have merged some steps and now it should look neater and more structural.